en/services/opendroneid.md

```diff
@@ -106,14 +110,19 @@ See below on how to combine data from other UAs.
 
 ### Heartbeat
 
-Each component listed in the table above, is required to regularly send out MAVLink [HEARTBEAT](../messages/common.md#HEARTBEAT) messages in order to facilitate discovery and monitoring of the component in the UAS.
+Each component involved in the drone ID MAVLink message exchange, is required to regularly send out MAVLink [HEARTBEAT](../messages/common.md#HEARTBEAT) messages in order to facilitate discovery and monitoring of the component in the UAS.
 For transceiver components (with component ids [MAV_COMP_ID_ODID_TXRX_1](../messages/common.md#MAV_COMP_ID_ODID_TXRX_1), [MAV_COMP_ID_ODID_TXRX_2](../messages/common.md#MAV_COMP_ID_ODID_TXRX_2), [MAV_COMP_ID_ODID_TXRX_3](../messages/common.md#MAV_COMP_ID_ODID_TXRX_3)), the `type` field in the HEARTBEAT message must be filled with [MAV_TYPE_ODID](../messages/common.md#MAV_TYPE_ODID).
 Please see further details in the [Heartbeat documentation](heartbeat.md).
- 
+
+The MAVLink [HEARTBEAT](../messages/common.md#HEARTBEAT) message also serves as the way for receiver components to identify the `systemid` of the GCS.
+The GCS will send out MAVLink [HEARTBEAT](../messages/common.md#HEARTBEAT) messages with its `systemid` field set to the GCS system ID and the type set to [MAV_TYPE_GCS](../message/common.md#MAV_TYPE_GCS).
+The receiver components shall interpret all MAVLink Open Drone ID messages from that system ID, as coming from the GCS.
+There is no dedicated component ID for GCSs, hence the system ID must be used instead for identifying the GCS.
```

Comment:
Technically it is the MAV_TYPE that is identifying the GCS, then mapped to system id. But this is OK.